BuiltInSpicyAnalyzer.cmake

```diff
@@ -38,9 +38,10 @@ function (spicy_add_analyzer)
             COMMENT "Compiling ${SPICY_ANALYZER_NAME} analyzer"
             COMMAND
                 ${CMAKE_COMMAND} -E env
-                "ZEEK_SPICY_LIBRARY_PATH=${PROJECT_SOURCE_DIR}/scripts/spicy" $<TARGET_FILE:spicyz>
-                -L ${spicy_SOURCE_DIR}/hilti/lib -L ${spicy_SOURCE_DIR}/spicy/lib -x
-                ${CMAKE_CURRENT_BINARY_DIR}/${NAME_LOWER} ${SPICYZ_FLAGS} ${SPICY_ANALYZER_SOURCES}
+                "ZEEK_SPICY_LIBRARY_PATH=${PROJECT_SOURCE_DIR}/scripts/spicy"
+                ASAN_OPTIONS=detect_leaks=0 $<TARGET_FILE:spicyz> -L ${spicy_SOURCE_DIR}/hilti/lib
```

Comment:
I'm setting `detect_odr_violation=0` in ASAN_OPTIONS to avoid an sqlite error I started to get.

Would `ASAN_OPTIONS="detect_leaks=0 ${ASAN_OPTIONS}` (or something similar work, too).

Reply:
Good point, added that in. I went with the documented way which uses `:` as a separator.